lib/turn.rb

```diff
@@ -0,0 +1,22 @@
+class Turn
+attr_reader :guess, 
+            :card, 
+            :feedback
+
+  def initialize(guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    @guess == card.answer
+  end
+
+  def feedback
+    if @guess == card.answer
+      'Correct!'
+    else 'Incorrect.'
+    end 
```

Comment:
While this works, ruby convention would have us put `'Incorrect.'` on its own line like this:
```ruby
...
else
  'Incorrect.'
end
```